VersionControlUI/src/DiffManager.cpp

```diff
@@ -94,9 +94,11 @@ DiffSetup DiffManager::initializeDiffPrerequisites(QString oldVersion, QString n
 
 	// load newer version
 	diffSetup.newVersionManager_ = createTreeManagerFromVersion(diffSetup.repository_, diffSetup.newVersion_);
+	diffSetup.newVersionManager_->setManagerName(project_+"#newVersion");
```

Comment:
So the manager name includes both the project and the version name? Why is that necessary? The project name is already in `manager->name()`?